packages/babel-plugin-transform-named-capturing-groups-regex/test/fixtures/runtime/.no-groups.js

```diff
@@ -0,0 +1,5 @@
+var re = /no-groups-\(?<looks-like-a-group>looks\)/;
```

Comment:
This test causes an infinite recursion in `regexp-tree`. I don't think that it should be a blocker, since irl regexps like this one are really rare.